templatefinder/utils.py

```diff
@@ -51,14 +55,24 @@ def find_all_templates(pattern='*.html'):
         ):
             loader_class = getattr(import_module(module), klass)
             if getattr(loader_class, '_accepts_engine_in_init', False):
```

Comment:
Maybe we could also inspect the signature of the `__init__` method on the `loader_class` to establish whether it accepts an `engine` or not? 

What do you think?